crates/cargo-test-support/src/lib.rs

```diff
@@ -319,6 +319,7 @@ impl Project {
         if let Some(ref mut p) = execs.process_builder {
             split_and_add_args(p, cmd);
         }
+        execs.env_remove("CARGO_TARGET_DIR");
```

Comment:
It's not clear, is this needed for something?

Reply:
Hmm, I thought I remembered this causing spurious test failures when I ran tests locally with a global `CARGO_TARGET_DIR` set, but I can't replicate the behavior. I'll drop this commit.